scripts/jest/utils/index.ts

```diff
@@ -26,7 +37,7 @@ function toMatchFile(receivedContent, filename) {
                 fs.unlinkSync(filename);
 
                 snapshotState.updated++;
-                return { pass: true, message: () => {} };
+                return { pass: true, message: () => {} } as CustomMatcherResult;
```

Comment:
It is correct the `message` function should return a `string` and not `void`.

Reply:
Ah, I was wondering about that. So should it return an empty string or something?

Reply:
Done. Empty string seems to work.